### PR TITLE
Privacy modal

### LINKED ISSUE #7978: Change privacy in the new builder
screen shot 2016-06-21 at 09 55 21

### PR DESCRIPTION
**Pull request by nobuti**
This PR fixes #7978.

privacy

cc @xavijam @piensaenpixel

### CODE REVIEW

**Comment by nobuti** [comment]
Rebased, added the loading and the trigger, and updated the screengif.

**Comment by xavijam** [comment]
:+1: 🚢 🇮🇹

File: lib/assets/javascripts/cartodb3/data/vis-definition-model.js [hunk 1]
```diff
@@ -18,6 +22,29 @@ module.exports = Backbone.Model.extend({
     var baseUrl = this._configModel.get('base_url');
     var version = this._configModel.urlVersion('visualization');
     return baseUrl + '/api/' + version + '/viz';
-  }
+  },
+
+  mapcapsURL: function () {
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb3/data/vis-definition-model.js]: mapcaps? this is from other PR right? :)

Comment (nobuti) [inline lib/assets/javascripts/cartodb3/data/vis-definition-model.js]: Ouch! Cleaning.

**Comment by saleiva** [comment]
VAMOOOOOS!

**Pull request merged**